src/markdown_generators.js

```diff
@@ -305,6 +325,15 @@ const generateResourceMarkdownForVideo = (media, courseData, pathLookup) => {
     }
   }
 
+  const parents = courseData["course_pages"].filter(
```

Comment:
Video files don't have a type in the parased.json file so we just get the parent_type and parent_title